openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/ExploreSortOrderFilter.spec.ts

```diff
@@ -68,18 +68,25 @@ test.describe('Explore Sort Order Filter', () => {
       await page.waitForLoadState('networkidle');
 
       await page.getByRole('button', { name: 'Data Assets' }).click();
-
       await page.waitForSelector(
         'data-testid="drop-down-menu" data-testid="loader"',
         {
           state: 'detached',
         }
       );
-
-      await page.waitForSelector(`[data-testid="${filter}-checkbox"]`, {
-        state: 'visible',
-      });
-      await page.getByTestId(`${filter}-checkbox`).check();
+      await page
+        .getByTestId('drop-down-menu')
+        .getByTestId('search-input')
+        .fill(filter.toLowerCase());
+      await page.waitForResponse('api/v1/search/aggregate?*');
```

Comment:
This isn’t the ideal way to wait for the API. Please check our codebase and follow the existing pattern.